internal/ignition/ignition_test.go

```diff
@@ -261,6 +263,20 @@ SV4bRR9i0uf+xQ/oYRvugQ25Q7EahO5hJIWRf4aULbk36Zpw3++v2KFnF26zqwB6
 			content = GetServiceIPHostnames("10.10.10.1,10.10.10.2")
 			Expect(content).To(Equal("10.10.10.1 assisted-api.local.openshift.io\n10.10.10.2 assisted-api.local.openshift.io\n"))
 		})
+		It("DHCP generation", func() {
```

Comment:
Can we add a test with the actual expected paths? Maybe it needs to be a subsystem test..
In order to verify we are actually creating a file in the `/etc/keepalived/unsupported-monitor.conf` path

Reply:
subsystem will be hard because we cannot test the files are actually created.
But a check for the path will be added here.